infoscreen/templates/infoscreen/base.html

```diff
@@ -7,76 +7,121 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    
-    
+
+
     <!-- JQuery first (3.5.0 breaks the site), then Popper.js, then Bootstrap JS -->
     <script src="https://code.jquery.com/jquery-3.4.1.js" integrity="sha256-WpOohJOqMqqyKL9FccASB9O0KwACQJpFTUBLTYOVvVU=" crossorigin="anonymous"></script>
     {# <script src="https://code.jquery.com/jquery-3.5.0.js" integrity="sha256-r/AaFHrszJtwpe+tHyNi/XCfMxYpbsRg2Uqn0x3s2zc=" crossorigin="anonymous"></script> #}
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.bundle.min.js" integrity="sha384-6khuMg9gaYr5AxOqhkVIODVIvm9ynTT5J4V1cfthmT+emCG6yVmEZsRHdxlotUnm" crossorigin="anonymous"></script>
     {# <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/2.4.0/cjs/popper.min.js" integrity="sha256-OH05DFHUWzr725HmuHo3pnuvUUn+TJuj8/Qz9xytFEw=" crossorigin="anonymous"></script> #}
     {# <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script> #}
-    
+
     <!-- Bootstrap CSS -->
     <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
-    
+
     <!-- Icon library -->
     <!-- Lineicons -->
     <link href="https://cdn.lineicons.com/2.0/LineIcons.css" rel="stylesheet">
     <!-- Font Awesome -->
     <script src="https://kit.fontawesome.com/19339791e2.js" crossorigin="anonymous"></script>
-    
+
     <!-- Pretty Checkbox -->
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/pretty-checkbox@3.0/dist/pretty-checkbox.min.css">
-    
+
     {# <link rel="shortcut icon" type="image/png" href="{% static 'root/img/favicon-32x32.png' %}"/> #}
-    
+
     <!-- flatpickr (requires .js file and init-script) -->
     <!-- Plugin for calendar date-picker -->
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/flatpickr/4.6.3/flatpickr.css" integrity="sha256-7vIWE+OHz3pLhuONuFqSa/Oh/YUUHFIMooHMwN1Q2bk=" crossorigin="anonymous" />
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/flatpickr/4.6.3/themes/dark.css" integrity="sha256-K7VO/dYboZOIjm2us6WE3Z8P8lsfUKaUVSnYYtPg6eE=" crossorigin="anonymous" />
-    
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+
```

Comment:
Vi har bootstrap fra før. Siden det er gammel versjon kan du vurdere å bytte ut. Undersøk gjerne de andre som er der også, JQuery f.eks.

Reply:
Jeg har ikke nevnt det for deg før, men har en måte å gjøre reviews på som jeg håper vi kan bli enige om. For min del ønsker jeg å få en kommentar på hva som ble gjort for å løse spørsmålet som ble tatt opp i tråden i stedet for å trykke resolve. Ofte kan det være så enkelt som `done` (det aller beste er å linke til en fixup commit). Jeg ønsker dette for å ha oversikt over hva som blir gjort samt slippe å åpne opp tråder som har kollapset. Husk å være kritisk til det jeg tar opp, er ikke alltid jeg har oversikt heller, en diskusjonstråd er bare bra. 
TL;DR: helst la eieren av en thread ha ansvaret for å trykke resolve.

Reply:
Hvis du lurer mer kan du bla gjennom noen av Magnus sine PR på rekenett.

Reply:
Tror denne er fikset ved å legge inn siste versjon av bootstrap, og fjerne tidligere versjon